src/coreclr/utilcode/util.cpp

```diff
@@ -566,7 +559,7 @@ BYTE * ClrVirtualAllocWithinRange(const BYTE *pMinAddr,
 /*static*/ CPU_Group_Info *CPUGroupInfo::m_CPUGroupInfoArray = NULL;
 /*static*/ LONG CPUGroupInfo::m_initialization = 0;
 
-#if !defined(FEATURE_NATIVEAOT) && (defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64))
+#if defined(TARGET_AMD64) || defined(TARGET_ARM64)
```

Comment:
Do we not need this for TARGET_LOONGARCH64 and TARGET_RISCV64 anymore?

Reply:
This block of code is under `HOST_WINDOWS` ifdef. We do not need to worry about LOONGARCH64 and RISCV64 for Windows.